index.html

```diff
@@ -2,232 +2,239 @@
+          <!--<a href='https://www.freepik.com/psd/woman'>Woman psd created by freepik - www.freepik.com</a>-->
+        </div>
+        <div class="col-lg-6  order-md-1 order-sm-2">
+          <h1 class="pop">The XR sandbox for your creativity</h1>
+          <h3 class="motto">Powered by a community-built library of 3D assets, music, animations, and effects</h3>
+
+        </div>
+
+      </div>
+    </div>
+    <div class="container value">
+      <div class="row">
+        <div class="col-lg-6">
+          <img class="demo" src="./images/alldevice2.png" width="400">
+        </div>
+        <div class="col-lg-6">
+          <h1 class="pop">Share and enjoy a universe of XR creations with any device</h1>
+          <h3 class="motto">Remove barriers between your XR artworks and your audience</h3>
+
+        </div>
+      </div>
+    </div>
+    <div class="end rounded shadow-lg">
+      <div class="container value mt-5">
+        <div class="row justify-content-center">
+          <div class="col-lg-9 center-close mt-5">
+            <h1 class="pop">Bring more people into contributing to the future of XR</h1>
+            <h3 class="motto">We envision building a mobile XR community with our users, welcoming everyone of all levels and backgrounds to share their creative story through art and music.
+            </h3>
+          </div>
+          <div class="col-lg-12">
+            <img style="width:inherit" src="./images/community.png">
+            <!--3D hands created by ThreeDee (https://www.handz.design/)-->
+          </div>
+
+        </div>
+      </div>
+    </div>
+
+    <div class="container value mt-5 ">
+      <div class="row justify-content-center pb-5">
+        <div class="col-lg-12 center">
+          <h1 class="pop"> Be one of the first to get the app </h1>
+        </div>
+        <div class="col-lg-4">
+          <form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
+            target="_blank" novalidate>
+            <div class="row justify-content-center">
+              <div class="form-group">
+                <input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
+                <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+              </div>
+              <button type="submit" class="btn btn-email btn-fill mt-4">NOTIFY ME</button>
+
+            </div>
+          </form>
+        </div>
+
+      </div>
+    </div>
+
+
+
+  </div>
 
+</body>
+<footer class="pt-5 pb-5 mt-5">
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-md">
+        <img class="mb-2" src="../images/logo-circle.png" alt="" width="24" height="24">
```

Comment:
This path has two dots instead of single like the others, so the image is not loading:
![image](https://user-images.githubusercontent.com/8660050/117749798-37572b80-b1c7-11eb-8251-884f5d28cb5e.png)

Reply:
weird it looks fine on my end